VideoLocker/src/main/java/org/edx/mobile/base/BaseSingleFragmentActivity.java

```diff
@@ -171,4 +166,10 @@ protected void hideMessageInSitu(){
         }
     }
 
+    private void showErrorDialog(String message) {
```

Comment:
Should this method be moved to `BaseFragmentActivity`, to be of more general use, like the `showErrorMessage()` and `showInfoMessage()` methods?